benches/reader.rs

```diff
@@ -0,0 +1,111 @@
+#![feature(test)]
+extern crate grcov;
+// extern crate rustc_hash;
+extern crate test;
+
+use grcov::{GCNO, GcovReaderBuf};
+use std::path::PathBuf;
+use test::{black_box, Bencher};
+
+
+#[bench]
+fn bench_reader_gcno(b: &mut Bencher) {
+    let mut gcno = GCNO::new();
+    b.iter(|| black_box(gcno.read(GcovReaderBuf::from("test/llvm/reader.gcno"))
```

Comment:
Here and elsewhere, we should read the file file first (with GcovReaderBuf::from), and then only benchmark gcno.read

Reply:
(This is still to do)